### PR TITLE
LIIKUNTA-688 | Askem dynamic loading fix

### PR DESCRIPTION
**Pull request by melniiv**
## Description
Refactored the component so that all hooks related to askem and consent are loaded dynamically

### Closes

**[LIIKUNTA-688](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-688):**

## Additional notes

[LIIKUNTA-688]: https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-688?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr748.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr748.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr748.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr748.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr748.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr748.dev.hel.ninja :laughing::tada::tada::tada:

File: packages/components/src/components/widgets/WidgetsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+type WidgetsContainerProps = {
+  appName: string;
+  hasFeedBack: boolean;
+  feedbackWithPadding: boolean;
+  consentUrl: string;
+  isModalConsent: boolean;
+  children?: React.ReactNode[];
+};
+
+export default function WidgetsContainer({
```
Comment (nikomakela) [inline packages/components/src/components/widgets/WidgetsContainer.tsx:16]: How about renaming this WidgetsContainer to "UserTrackingFeatures" or something like that? Now the name feels too generic. It makes me want to use it for every widget :)

**Review by nikomakela — APPROVED** [review]

File: packages/components/src/components/widgets/WidgetsContainer.tsx (hunk 1 above)
Comment (nikomakela) [reply packages/components/src/components/widgets/WidgetsContainer.tsx:16]: I don't like that a "widget container" tightly couples the Askem and CookieConsent together by jsut hard coding them. The WidgetsContainerProps are just props that it's hard coded sub components are using, so should the field types be linked from the sub components?
```
type WidgetsContainerProps = {
  appName: string;
  hasFeedBack: boolean;
  feedbackWithPadding: boolean;
  consentUrl: string;
  isModalConsent: boolean;
  children?: React.ReactNode[];
};
```

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr748.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr748.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr748.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr748.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr748.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr748.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**